<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateStudenti</class>
 <widget class="QDialog" name="updateStudenti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>663</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>663</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Studenti</string>
  </property>
  <widget class="QLineEdit" name="telefonEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>  JUDET:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FACULTATE:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localitateEdit">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>310</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DOMICILIU:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LOCALITATE:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="domiciliuEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>211</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelInsert">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NUME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMAIL:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="facultateBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okInsert">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TELEFON:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="judetBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="nrMatricolBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Numar matricol:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
